katcp/server.py

```diff
@@ -34,9 +34,11 @@
 from .sampling import format_inform_v5, format_inform_v4
 from .core import (SEC_TO_MS_FAC, MS_TO_SEC_FAC, SEC_TS_KATCP_MAJOR,
                    VERSION_CONNECT_KATCP_MAJOR, DEFAULT_KATCP_MAJOR)
-from .version import VERSION, VERSION_STR
 from .kattypes import (request, return_reply)
 
+from katversion import get_version
```

Comment:
Ah that makes sense, I was a bit unsure of how to handle it, this explains it nicely. I'm a bit of a noob when it comes to packaging + versioning, but I'm learning 👍